packages/components/app/styles/components/button.scss

```diff
@@ -96,6 +92,14 @@
   @include hds-button-color-critical();
 }
 
+// ISINLINE
+.hds-button--is-inline {
+  display: inline-block;
+}
+
+// SIZE
+@include hds-button-size-classes("hds-button");
```

Comment:
nitpick: probably I would have the mixin _before_ the display override, in case in the future the mixin includes `display` declarations.